.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 
 default_language_version:
-  python: python3.7
+  python: python3
```

Comment:
It looks like `default_language_version` is optional - can we just remove it? https://pre-commit.com/#top_level-default_language_version

Reply:
That's probably a better approach, just pushed a commit doing so.